src/Config/HostsFile.php

```diff
@@ -77,6 +77,8 @@ public static function loadFromPathBlocking($path = null)
      * Instantiate new hosts file with the given hosts file contents
      *
      * @param string $contents
+     *
+     * @internal
```

Comment:
Should we really mark (parts of) this class as internal? I can see some use cases for this outside of this package, so I think we might be better off keeping this as part of our API without an in-depth discussion. The other extreme might be to mark everything as internal that is not covered in our documentation, but I'd rather not have this discussion right now. What do you think?

Reply:
Sure, but only the static calls have been documented to be honest. Everything after the instantiation hasn't. Will remove the `@internal`'s